src/msg.c

```diff
@@ -391,50 +378,14 @@ static bool property_changed(const char *name,
 	struct l_queue *list;
 	struct session *session;
 	struct knot_device *device;
-	struct mydevice *mydevice;
 	char id[KNOT_ID_LEN];
 
 	/* FIXME: manage link overload or not connected */
 	session = l_queue_find(session_list, session_node_fd_cmp, user_data);
 	if (!session)
 		return false;
 
-	/* FIXME: Memory leak & detect if schema has changed */
-	if (strcmp("schema", name) == 0) {
-		if (session->schema && strcmp(session->schema, value) == 0)
```

Comment:
Since we are no longer using `session->schema` remove it from struct

Reply:
Done.